packages/forms/frontend/sirius-components-forms/src/form/FormEventFragments.types.ts

```diff
@@ -283,6 +283,12 @@ export interface GQLButton extends GQLWidget {
   style: GQLButtonStyle;
 }
 
+export interface GQLSplitButton extends GQLWidget {
+  buttonLabel: string | null;
```

Comment:
It seems there is an inconsistency for `buttonLabel` and `imageURL` fields on SplitButton.
Sometimes they are declared (here, or in form.graphqls) but not declared in the `form.ecore` or in `org.eclipse.sirius.components.view.form.SplitButtonDescription.java`.
I think it should be removed everywhere (please verify), but could you check that and make it all consistent please?